### PR TITLE
RMB-686: Fix documentation about postTenant error handling

### PR DESCRIPTION
**Pull request by julianladisch**
Use `handler.handle(res);` to return HTTP status provided by
RMB TenantAPI to comply with the API specs: 200 for upgrade, 201 for install

Use unabbreviated variables names to comply with coding standards
and for readability.

Add missing `@Validate`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1356,30 +1370,29 @@ as resources and as JSON files, you can use the TenantLoading utility.
...
-      .withIdContent().
+      .withIdContent()
       .add("data1", "instances")
       .add("data2", "items");
-    tl.perform(ta, headers, vertx, res1 -> {
+    tl.perform(tenantAttributes, headers, vertx, res1 -> {
       if (res1.failed()) {
-        hndlr.handle(io.vertx.core.Future.succeededFuture(PostTenantResponse
-          .respond500WithTextPlain(res1.cause().getLocalizedMessage())));
+        hndlr.handle(Future.succeededFuture(PostTenantResponse
```
Comment (hjiebsco) [inline README.md:1389]: hndlr -> handler?

Comment (julianladisch) [reply README.md]: Yes, thanks for catching.

File: README.md [hunk 2]
```diff
@@ -1308,44 +1308,58 @@ import javax.ws.rs.core.Response;
...
 If you wish to load data for your module, that should be done after the DB has been successfully initialized,
 e.g. do something like:
+
 ```
-public void postTenant(TenantAttributes ta, Map<String, String> headers,
-  super.postTenant(ta, headers, res -> {
-    if (res.failed()) {
-      hndlr.handle(res);
+public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers,
+  super.postTenant(tenantAttributes, headers, res -> {
```
Comment (hjiebsco) [inline README.md:1346]: Missing a line above super.postTenant
`Handler<AsyncResult<Response>> handler, Context context) {`

File: README.md [hunk 3]
```diff
@@ -1356,30 +1370,29 @@ as resources and as JSON files, you can use the TenantLoading utility.
 ```java
 import org.folio.rest.tools.utils.TenantLoading;
 
-public void postTenant(TenantAttributes ta, Map<String, String> headers,
-  super.postTenant(ta, headers, res -> {
-    if (res.failed()) {
-      hndlr.handle(res);
+public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers,
+  super.postTenant(tenantAttributes, headers, res -> {
```
Comment (hjiebsco) [inline README.md:1374]: Same. Missing a line above super.postTenant
`Handler<AsyncResult<Response>> handler, Context context) {`

File: README.md (hunk 2 above)
Comment (julianladisch) [reply README.md:1346]: `res -> { ... }` is `Handler<AsyncResult<Response>> handler`, 
`context` in line 1362 is `Context context`.

Comment (julianladisch) [reply README.md:1349]: Thanks, good catch, fixed.

File: README.md (hunk 3 above)
Comment (julianladisch) [reply README.md:1380]: Thanks, good catch, fixed.

**Pull request merged**